src/node_sqlite.cc

```diff
@@ -67,6 +67,63 @@ using v8::Value;
     }                                                                          \
   } while (0)
 
+#define SQLITE_VALUE_TO_JS(                                                    \
+    from, unreachable_message, isolate, use_big_int_args, ...)                 \
+  switch (sqlite3_##from##_type(__VA_ARGS__)) {                                \
```

Comment:
I would wrap the whole macro in a `do...while` loop (like the others in this file).

Reply:
done